libraries/helpers.rb

```diff
@@ -153,3 +153,10 @@ def ami_bootstrapped?
 
   'aws-parallelcluster-' + node['cfncluster']['cfncluster-version'] == version && node['cfncluster']['skip_install_recipes'] == 'yes'
 end
+
+#
+# Check if this is an ARM instance
+#
+def arm_instance?
+  node['kernel']['machine'] == 'aarch64'
```

Comment:
is this value common across all the supported platforms (e.g. for Ubuntu)?

Reply:
It's worked so far in my testing.